src/components/ec-main-container/ec-main-container.vue

```diff
@@ -1,8 +1,16 @@
 <template>
-  <div class="ec-main-container">
+  <div class="tw-p-24 tw-min-h-full ec-main-container">
```

Comment:
taking into account we keep the `. ec-main-container` class, wouldn't it be interesting to move these utility classes with `@apply` ?